src/content/release-notes/load-balancing.yaml

```
@@ -5,6 +5,10 @@ productLink: "/load-balancing/"
 productArea: Application performance
 productAreaLink: /fundamentals/reference/changelog/performance/
 entries:
+  - publish_date: "2025-03-13"
+    title: Update to Load Balancing Analytics
```

Since the docs don't capitalize the 'A' in LB analytics, perhaps we should leave this lowercase?
```suggestion
    title: Update to Load Balancing analytics
```